code_generation/producer.py

```diff
@@ -45,12 +45,13 @@ def writecall(self, config, scope, shift=""):
             if isinstance(self.output, list)
             else self.output.get_leaf(shift, scope)
         )
-        config[shift]["input"] = ",".join(
+        config[shift]["input"] = '", "'.join(
```

Comment:
better put quotes around it here (as for input_coll below) and remove them from the calls in the producer definitions in order to have complete strings returned.

Reply:
I think a similar update could be made for output, but I think that's not necessary for this PR

Reply:
and then remove it from the definition of of the producer call ?

Reply:
yes